.pre-commit-config.yaml

```diff
@@ -43,6 +43,7 @@ repos:
       - id: check-json
       - id: no-commit-to-branch
         args:
+          - --branch=dev
```

Comment:
@springstan 
What is the recommended way for me to push to my dev branch without having to change this?
Should i just not have a 'dev' branch, and name it something else in the future?

Reply:
> Should i just not have a 'dev' branch, and name it something else in the future?

Yes indeed.